README.md

```diff
@@ -97,6 +97,22 @@ You can optionally attach a configuration file to the container by adding the
 
 **Note**: The recommended restart policy for the container is `on-failed` or `no`
 
+#### Docker Compose Example
+```yaml
+...
+  auto-southwest:
+    image: jdholtz/auto-southwest-check-in
+    container_name: auto-southwest
+    networks:
```

Comment:
Is `networks:` and `security_opt:` necessary for you? I removed it and it still worked perfectly fine.

Reply:
You are totally correct and it cleans it up a little. Address in: https://github.com/jdholtz/auto-southwest-check-in/pull/171/commits/c43f49190eb9a96cfbb1f864ba653b94c246bb48